src/bio/bio_xstream.c

```diff
@@ -83,18 +83,27 @@ bio_spdk_env_init(void)
 
 	spdk_env_opts_init(&opts);
 	opts.name = "daos";
+
 	/*
 	 * TODO: Set opts.mem_size to nvme_glb.bd_mem_size
 	 * Currently we can't guarantee clean shutdown (no hugepages leaked).
 	 * Setting mem_size could cause EAL: Not enough memory available error,
 	 * and DPDK will fail to initialize.
 	 */
 
-	if (nvme_glb.bd_shm_id != DAOS_NVME_SHMID_NONE)
+	if (nvme_glb.bd_shm_id == DAOS_NVME_SHMID_NONE)
```

Comment:
I don't follow this change. The opts.shm_id is already initialized as -1 above.

Reply:
fixed, thanks